### DISCUSSION TITLE
Is mobx(5*) + mobx-react (6*) versions are compatable for react 18 version

### DISCUSSION THREAD

**Discussion by Lingareddyvinitha** [mobxjs/mobx#3614] [Q&A]
[From mobx-react official doc](https://github.com/mobxjs/mobx/tree/main/packages/mobx-react#mobx-react) resource as a guide. We concluded that mobx(5*) + mobx-react (6*) versions are compatible for react 18 version. But after updating our repo we got some issues related to [React 18: observer not working with React Class components <React.StrictMode>](https://github.com/mobxjs/mobx/issues/3395)

Our Versions
```
"mobx": "5.15.0",
"mobx-react": "6.1.4",
"react": "18.2.0",
  
  ```

Are mobx(5*) and mobx-react(6*) versions compatible with react 18? 
If not, What are the compatible versions of mobx and mobx-react that can be used when using React 18?

[mweststrate](https://github.com/mweststrate), [urugator](https://github.com/urugator)

**Comment by urugator** [comment]
We don't have tests for older mobx versions + React v18, therefore we don't know how much incompatible they actually are.
Could be "just" `<StrictMode>` and concurrent features, could be more. Updated the docs. Sorry for the inconvenience.
